RecoTracker/IterativeTracking/python/PixelLessStep_cff.py

```diff
@@ -201,11 +201,15 @@
                 FilterStripHits    = cms.bool(True),
                 ClusterShapeHitFilterName = cms.string('pixelLessStepClusterShapeHitFilter'),
                 ClusterShapeCacheSrc      = cms.InputTag('siPixelClusterShapeCache') # not really needed here since FilterPixelHits=False
-            ), 
-            _StripSubClusterShapeSeedFilter.clone()
+            )
         )
     )
 )
+
+from RecoPixelVertexing.PixelLowPtUtilities.StripSubClusterShapeSeedFilter_cfi import StripSubClusterShapeSeedFilter as _StripSubClusterShapeSeedFilter
+from Configuration.ProcessModifiers.approxSiStripClusters_cff import approxSiStripClusters
+(~approxSiStripClusters).toModify(pixelLessStepSeeds.SeedComparitorPSet.comparitors, func = lambda list: list.append(_StripSubClusterShapeSeedFilter.clone()) )
```

Comment:
Should this modification go into RecoTracker/IterativeTracking/python/PixelLessStep_cff.py?

Reply:
Sorry Andrea, I miss the point somehow.

Reply:
this modification is *already* in `RecoTracker/IterativeTracking/python/PixelLessStep_cff.py`, so I don't understand the point either.

Reply:
My bad! Sorry for the noise here @mandrenguyen and @mmusich